File: BE/tipsy/core-web/src/main/java/com/ssafy/coreweb/dto/TokenDto.java
package com.ssafy.coreweb.dto;

import java.util.Date;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;

@AllArgsConstructor
@Getter
@Builder
public class TokenDto {
	private String accessToken;
	private String refreshToken;
	private Date accessTokenExpiresIn;
	private String Authority;
}

File: Android/app/src/main/java/com/hanjan/hanjangame/dto/TokenDto.kt
package com.hanjan.hanjangame.dto

import com.fasterxml.jackson.annotation.JsonProperty

data class TokenDto(
    @JsonProperty("accessToken")
    val accessToken: String,
    @JsonProperty("accessTokenExpiresIn")
    val accessTokenExpiresIn: String,
    @JsonProperty("authority")
    val authority: String?,
    @JsonProperty("refreshToken")
    val refreshToken: String
)